jetty-ee10/pom.xml

```diff
@@ -69,6 +69,11 @@
 
   <dependencyManagement>
     <dependencies>
+      <dependency>
```

Comment:
These lines are duplicates of lines 77-81.

Reply:
Duplicate lines removed.